.github/workflows/reusable-ubuntu.yml

```
@@ -16,6 +16,7 @@ jobs:
     env:
       OPENSSL_VER: 3.0.13
       PYTHONSTRICTEXTENSIONBUILD: 1
+      FORCE_COLOR: 1
```

Nit: alphabetise, makes it easier for when it grows:
```suggestion
      FORCE_COLOR: 1
      OPENSSL_VER: 3.0.13
      PYTHONSTRICTEXTENSIONBUILD: 1
```